### ISSUE TITLE
meeting on Dec 12

### ISSUE THREAD

**Issue by TanyaNikitina** [KatyaAplonova/ReportedDiscourseProject#23]
It seems very likely that we will have a project meeting on Dec 12. We will certainly have other meetings before that but cannot yet schedule them. I just thought I should let you know about Dec 12 since I am already aware of it. Please integrate into the calendar.

**Comment by rvoll** [comment]
Good to know! Thanks for the long notice.

Rebecca

**Comment by IndianaTones** [comment]
Hi all,

I have some slides planned to present to you regarding the pronominal mismatches in Ben Tey Dogon.

@KatyaAplonova I believe also has something similar planned...?

**Comment by KatyaAplonova** [comment]
Yep! I have an unfinished abstract to discuss about serial verb-like construction in discourse reports.
